Software-Engineering.md

```diff
@@ -133,6 +133,24 @@ _We avoid distinguishing too much between the Junior Software Engineer and Softw
 - I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
 - I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
 
+## Principal Software Engineer
+
+> I am an influential Engineering [leader](Leadership.md) who identifies problems and opportunities relevant to Octopus' strategy, builds consensus to invest in solutions, and guides Engineering to deliver significant lasting change across Octopus.
+
+### I have a knack for uncovering important problems and opportunities that are critical to the long-term success of Octopus
+
+- I have a deep understanding of Octopus products and our strategy
+- I identify significant problems, inhibitors and opportunities, and advocate for solutions that align our strategy and Engineering efforts
+- I collaborate with others to articulate solutions, positively influence and convince that we should fund their implementation
```

Comment:
I think there are two things here - maybe we can say them both explicitly?
- Principal engineers have a good track record of identifying the important problems and solutions that tend to be funded
- Principal engineers are good at persuading the organisation/betting table/etc. to fund these pitches/missions/etc.

Reply:
I've improved part of the above, but I think there is an item missing regarding working with product that your second point helps steer us toward. I'll see where is best to add that item.